pre_commit_python_hooks/check_latest_in_module_source.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+import argparse
+
+
+PASS = 0
+FAIL = 1
+
+
+def main(argv: Sequence[str] | None = None) -> int:
+    ISSUE = ""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("filenames", nargs="*", help="Filenames to fix")
+    args = parser.parse_args(argv)
+    # print(args)
+
+    for arg in args.filenames:
+        with open(arg, "r+") as file:
+            lines = file.readlines()
+
+            for index, line in enumerate(lines):
+
+                if "#ignore" in line:
```

Comment:
Could we use `# tf:ignore` to make sure we understand the perimeter of the instruction

Reply:
Maybe it's worth adding the pragma in the README in a section which does not exist yet like `Hooks Documentation`

Reply:
ignore has been update to # tf:ignore